src/services/helpers.ts

```diff
@@ -194,12 +194,19 @@ export const getSortedAndFilteredVocabularyItems = (
   vocabularyItems: VocabularyItem[] | null,
   searchString: string
 ): VocabularyItem[] => {
+  const collator = new Intl.Collator('de-De', { sensitivity: 'base', usage: 'sort' })
+
   const normalizedSearchString = normalizeSearchString(searchString)
 
+  const getNouns = (item: { word: string }) => {
+    const words = item.word.split(' ')
+    return words.find((word: string) => word[0] === word[0].toUpperCase()) as string
```

Comment:
This `as string` cast is needed, because if no word matches your find function the function retruns nothing. It would be better to do:
`return words.find((word: string) => word[0] === word[0].toUpperCase()) ?? word`
as if there is some error or word without a noun in the words, e.g. "der helm", it would just return "helm" for sorting instead of nothing. This should not happen, but it is possible, so we should also think of this case.